### ISSUE TITLE
StatamicRadPack  \  Runway  \  Exceptions  \ There are no fields defined in the orders blueprint.

### ISSUE THREAD

**Issue by deathlylol** [statamic-rad-pack/runway#586]
### Description

I use simple-commerce for my statamic app. I implemented [database orders](https://simple-commerce.duncanmcclean.com/database-orders#content-the-model). But I get an error 
![image](https://github.com/user-attachments/assets/a14dd84e-a2ea-45f7-b725-ce6eea7f09b5)

This is my runway.php

```
'resources' => [
        \DuncanMcClean\SimpleCommerce\Customers\CustomerModel::class => [
            'name' => 'Customers',
            'handle' => 'customers',
            'hidden' => true,
        ],

        \DuncanMcClean\SimpleCommerce\Orders\OrderModel::class => [
            'name' => 'Orders',
            'handle' => 'orders',
            'hidden' => true,
            'read_only' => true,
        ],
    ],
```
simple-commerce.php

```
 'content' => [
        'customers' => [
            'repository' => \DuncanMcClean\SimpleCommerce\Customers\EloquentCustomerRepository::class,
            'model' => \DuncanMcClean\SimpleCommerce\Customers\CustomerModel::class,
        ],

        'orders' => [
            'repository' => \DuncanMcClean\SimpleCommerce\Orders\EloquentOrderRepository::class,
            'model' => \DuncanMcClean\SimpleCommerce\Orders\OrderModel::class,
        ],

        'products' => [
            'repository' => \DuncanMcClean\SimpleCommerce\Products\EntryProductRepository::class,
            'collection' => 'products',
        ],
    ],
```

### Steps to reproduce

I did every step from this page - https://simple-commerce.duncanmcclean.com/database-orders#content-the-model

### Environment

php please support:details

Environment
Application Name: Statamic
Laravel Version: 11.20.0
PHP Version: 8.3.1
Composer Version: 2.7.7
Environment: local
Debug Mode: ENABLED
URL: bon_core.test
Maintenance Mode: OFF
Timezone: UTC
Locale: en

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: file
Database: mysql
Logs: stack / single
Mail: log
Queue: sync
Session: file

Simple Commerce
Currencies: GBP
Gateways: Dummy
Repository: Customer: DuncanMcClean\SimpleCommerce\Customers\EloquentCustomerRepository
Repository: Order: DuncanMcClean\SimpleCommerce\Orders\EloquentOrderRepository
Repository: Product: DuncanMcClean\SimpleCommerce\Products\EntryProductRepository
Shipping Methods: Free Shipping
Tax Engine: DuncanMcClean\SimpleCommerce\Tax\Standard\TaxEngine

Statamic
Addons: 3
Sites: 1
Stache Watcher: Enabled (auto)
Static Caching: Disabled
Version: 5.22.0 Solo

Statamic Addons
duncanmcclean/simple-commerce: 7.4.3
statamic-rad-pack/runway: 7.8.0
statamic/eloquent-driver: 4.12.3

Statamic Eloquent Driver
Asset Containers: eloquent
Assets: eloquent
Blueprints: eloquent
Collection Trees: eloquent
Collections: eloquent
Entries: eloquent
Forms: eloquent
Global Sets: eloquent
Global Variables: eloquent
Navigation Trees: eloquent
Navigations: eloquent
Revisions: file
Sites: eloquent
Taxonomies: eloquent
Terms: eloquent
Tokens: eloquent

**Comment by duncanmcclean** [comment]
You'll be getting that since the Order & Customer blueprints are empty. 

Simple Commerce should have copied them as part of one of the commands but it obviously hasn't, so you'll need to update the blueprints manually:

* [`resources/blueprints/vendor/runway/order.yaml`](https://github.com/duncanmcclean/simple-commerce/blob/7.x/src/Console/Commands/stubs/runway_order_blueprint.yaml)
* [`resources/blueprints/vendor/runway/customer.yaml`](https://github.com/duncanmcclean/simple-commerce/blob/7.x/src/Console/Commands/stubs/runway_customer_blueprint.yaml)

**Comment by deathlylol** [comment]
@duncanmcclean is there any separate command for such cases?

**Comment by duncanmcclean** [comment]
No, you can just copy the contents of the files manually into your project.

**Comment by deathlylol** [comment]
@duncanmcclean but I have this files in my project 
![image](https://github.com/user-attachments/assets/466023af-5bc9-4182-9d4b-cc6e3179921a)

**Comment by duncanmcclean** [comment]
> I did it, but it didn't help me 😅

Is it still giving you the same error? A different error? Can you take a screenshot of the `orders.yaml` file in your IDE/code editor.

**Comment by duncanmcclean** [comment]
Oh, you're storing blueprints in the database. You'll need to copy the contents of the blueprints into the database. 

Alternativley, you could export your blueprints from the database back to files, then re-import them again.

**Comment by deathlylol** [comment]
Yes! So I need to return everything to the file format, and then to the database?

**Comment by duncanmcclean** [comment]
Yeah, that's probably the easiest way.
